03-Bootstrap/assignment/faq.html

```diff
@@ -0,0 +1,163 @@
+                                    data-bs-target="#collapseThree" aria-expanded="false"
+                                    aria-controls="panelsStayOpen-collapseThree">
+                                    Can you pack my fish and trophies home?
+                                </button>
+                            </h2>
+                            <div id="collapseThree" class="accordion-collapse collapse">
+                                <div class="accordion-body">
+                                    <strong>We will take them to our personal houses afterward.</strong>
+                                    <p>Fish can be packaged in a variety of ways, including polyethylene or
+                                        polypropylene film, polyethylene bags, plastic bags in cartons, waxed paper
+                                        boxes, or corrugated fiberboard cartons.
+                                        Dry ice
+                                        If using dry ice, the package must allow for carbon dioxide gas to be released,
+                                        and it must be labeled as "Carbon Dioxide Solid" or "Dry Ice". The package must
+                                        also include the net weight of the dry ice. The TSA limits dry ice to 5 pounds
+                                        for both carry-on and checked baggage</p>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+
+                </div>
+
+            </div>
+            <div class="row text-light bg-dark text-center"><!-- footer with nav links -->
+                <h4 class="p-4 m-3">FishCamp LLC - Sustaining Natural Memories</h4>
+                <p class="lead p-4">Charters and guided and backpack fishing sustainably in WA, OR, ID mountain
+                    lakes
+                    and
+                    streams</p>
+                <nav class="navbar navbar-dark btn-group-vertical justify-content-around">
+                    <a href="./index.html"><button class="btn btn-outline-primary m-3">Landing Page</button></a>
+
+                    <a href="./booking.html"><button class="btn btn-outline-primary m-3">Book A Trip</button></a>
+
+                    <a href="./contact.html"><button class="btn btn-outline-primary m-3">Contact Us</button></a>
+
+                    <a href="./gallery.html"><button class="btn btn-outline-primary m-3">Trip Gallery</button></a>
+
+                </nav>
+                <p class="p-4">Created 2024 Sites-To-Go LLC Seattle</p>
+            </div>
+        </div>
+    </main>
+    <!-- popper and jquery links -->
+
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.8/dist/umd/popper.min.js"
+        integrity="sha384-I7E8VVD/ismYTF4hNIPjVp/Zjvgyol6VFvRkX/vR+Vc4jQkC+hVqc2pM8ODewa9r"
+        crossorigin="anonymous"></script>
+
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
```

Comment:
Please don't use jQuery. Everything we can do we can do with bootstrap and/or vue in the upcoming assignments. jQuery is very old school and we should avoid it.